c3po/api/controller/data_controller.py

```diff
@@ -0,0 +1,22 @@
+from flask import abort, request
+from flask_restx import Namespace, Resource, reqparse
+
+from c3po.api.dto import DataDto
+from c3po.api.service.data_service import DataService
+
+data_ns = DataDto.ns
+
+parser = reqparse.RequestParser()
+
+
+@data_ns.route("/post")
```

Comment:
We should add some very simple header check for this request with some secret value so creative or naive people don't send random data to this endpoint.